src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends AbstractFigure {
```

Comment:
Avoid using abstract classes to set behavior for classes. Consider using an interface to define behavior and use composition to share common code if necessary.